sqlmesh/core/model/definition.py

```diff
@@ -1428,7 +1428,6 @@ def is_breaking_change(self, previous: Model) -> t.Optional[bool]:
                 this_query,
                 matchings=[(previous_query, this_query)],
                 delta_only=True,
-                copy=False,
```

Comment:
How confident are we that this is a good idea?

Reply:
this flag doesn't exist anymore. diff will always copy now if it's necessary, and not copy when it's not. it does this by checking for overlapping / duplicate object ids